webhook/resource_admission_controller.go

```diff
@@ -59,18 +59,22 @@ type ResourceAdmissionController struct {
 	handlers map[schema.GroupVersionKind]GenericCRD
 
 	disallowUnknownFields bool
+	WithContext           func(context.Context) context.Context
```

Comment:
does this need to be public?

Reply:
Not currently, but when this moves to its own package, the test update I made will need this to be public 🙄

Reply:
added a comment